test/test-ipc-send-recv.c

```diff
@@ -376,7 +376,8 @@ int run_ipc_send_recv_helper(uv_loop_t* loop, int inprocess) {
     r = uv_listen((uv_stream_t*)&ctx2.listen, SOMAXCONN, listen_cb);
     ASSERT(r == 0);
   } else {
-    r = uv_pipe_open(&ctx2.channel, 0);
+    uv_os_fd_t stdin_handle = uv_convert_fd_to_handle(0);
```

Comment:
It would work, since for the purposes of this test harness they happen to get set to the same stream. However, testing with `fd = 0` is slightly more accurate – and makes for better example code for the expected nodejs usage.